src/libData/AccountData/AccountStore.cpp

```diff
@@ -213,17 +213,24 @@ bool AccountStore::DeserializeDelta(const bytes& src, unsigned int offset,
   }
 
   if (revertible) {
+    if (LOOKUP_NODE_MODE) {
```

Comment:
may not need this mech in case of revertible. `revertible` is true only for miner nodes.